plugins/woocommerce-admin/client/products/sections/attributes-section.tsx

```diff
@@ -43,11 +42,7 @@ export const AttributesSection: React.FC = () => {
 				</>
 			}
 		>
-			<Card>
-				<CardBody>
-					<AttributeField { ...getInputProps( 'attributes' ) } />
-				</CardBody>
-			</Card>
+			<AttributeField { ...getInputProps( 'attributes' ) } />
```

Comment:
Instead of passing all input props here, it would be clearer within the `<AttributeField />` component if we just move the `useContext` hook inside. Of course you could make the case that this is a bit more flexible, so entirely optional if we see other use cases that could benefit.

Reply:
Yeah, I thought that may be the case. I would say that it's likely that an input field would normally appear inside of a form in some manner, but I understand the value of treating this as a standard input. Sounds good 👍🏻